Formula/stgit.rb

```diff
@@ -12,18 +14,33 @@ class Stgit < Formula
     sha256 "a8c5a52941bb5c524f97bddf295dbf65b79ec74b4ec5a0d0ebcdb25429e1e03d" => :high_sierra
   end
 
+  depends_on "asciidoc" => :build
+  depends_on "xmlto" => :build
+  depends_on "python@3.8"
+
   def install
-    ENV["PYTHON"] = "python" # overrides 'python2' built into makefile
+    ENV["PYTHON"] = "python3"
```

Comment:
This should be `Formula["python@3.8"].opt_bin/"python3"` as python3 can be anything we have no control over.